src/python/ensembl/io/genomio/manifest_stats.py

```diff
@@ -17,70 +17,91 @@
 
 import gzip
 import io
-import json
 from pathlib import Path
 from shutil import which
 from statistics import mean
 import subprocess
 from typing import Dict, List, TextIO
-
+import argparse, os
 from BCBio import GFF
-import eHive
+import json
+
 
-from ensembl.brc4.runnable.utils import get_json
+def get_args():
+    """
+    Handle script arguments
+    :return: Script arguments
+    """
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-m', '--manifest_dir', help='Manifest file path', type=str, required=True)
+    parser.add_argument('-a', '--accession', help='Accession_ID', type=str, required=True)
+    parser.add_argument('-e', '--error', help='error status, True or False, Default False', default="false")
+    parser.add_argument('-dsin', '--datasets_bin', help='Datasets_bin Status', type=str, required=True)
 
 
-class manifest_stats(eHive.BaseRunnable):
+    args = parser.parse_args()
+    return args
+
+class manifest_stats:
+    def __init__(self, manifest, accession, error, datasets_bin):
+        self.manifest = f'{manifest}/manifest.json'
+        self.accession = accession
+        self.error = error
+        self.datasets_bin = datasets_bin
+        self.manifest_parent = manifest
 
     def param_defaults(self):
         return {
-            "datasets_bin" : "datasets",
+            "datasets_bin": "datasets",
         }
 
     def run(self):
-        manifest_path = Path(self.param_required("manifest"))
-        manifest = self.get_manifest(manifest_path)
-        
-        stats = [self.param("accession")]
-
-        self.param("error", False)
-        if "gff3" in manifest:
-            stats += self.get_gff3_stats(Path(manifest["gff3"]))
-        if "seq_region" in manifest:
-            stats += self.get_seq_region_stats(Path(manifest["seq_region"]))
-        
-        stats_path = manifest_path.parent / "stats.txt"
+        manifest = self.get_manifest()
+
+        stats = [self.accession]
+
+        if self.error == 'false':
```

Comment:
self.error is boolean, not a string:
```suggestion
        if not self.error:
```